integration_tests/integration_test.py

```diff
@@ -84,7 +85,7 @@ def setUp(self):
         rest_communication.post_entry(
             'samples',
             {'library_id': self.library_id, 'project_id': self.project_id, 'sample_id': self.sample_id,
-             'run_elements': [e['run_element_id'] for e in run_elements], 'required_yield': 100000000}
+             'run_elements': [e['run_element_id'] for e in run_elements], 'required_yield_q30': 900000000}
```

Comment:
Not sure why we're setting the `required_yield_q30` here instead of the `required_yield`